bin/MadGraph5_aMCatNLO/cards/production/2017/13TeV/VVVEFT/Dim8/WWZ/WWZ_ScaleST_proc_card.dat

```diff
@@ -0,0 +1,19 @@
+set default_unset_couplings 99
+set loop_color_flows False
+set gauge unitary
+set max_npoint_for_channel 0
+set group_subprocesses Auto
+set ignore_six_quark_processes False
+set loop_optimized_output True
+set complex_mass_scheme False
+import model QCKM_5_Aug21v2
+define p = g u c d s u~ c~ d~ s~
+define j = g u c d s u~ c~ d~ s~
```

Comment:
these two lines can be removed

Reply:
Hi @menglu21 thanks! By default Madgraph assumes the proton includes a `b`?

Reply:
no, cuz you define  "p = g u c d s b u~ c~ d~ s~ b~" in the next lines, so these two lines will be overwritten

Reply:
Thanks, Meng! We have decided to go with the 4 flavor scheme. So, let me update the PR with those cards.